Documentation/ceph-fs-mirror-crd.md

```diff
@@ -39,8 +39,61 @@ If any setting is unspecified, a suitable default will be used automatically.
 
 ### FilesystemMirror Settings
 
+* `peers`: to configure mirroring peers
+  * `secretNames`:  a list of peers to connect to. Currently (Ceph Pacific release) **only a single** peer is supported where a peer represents a Ceph cluster.
+  However, if you want to enable mirroring of multiple filesystems, you would have to have **one Secret per filesystem**.
 * `placement`: The cephfs-mirror pods can be given standard Kubernetes placement restrictions with `nodeAffinity`, `tolerations`, `podAffinity`, and `podAntiAffinity` similar to placement defined for daemons configured by the [cluster CRD](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/cluster.yaml).
 * `annotations`: Key value pair list of annotations to add.
 * `labels`: Key value pair list of labels to add.
 * `resources`: The resource requirements for the cephfs-mirror pods.
 * `priorityClassName`: The priority class to set on the cephfs-mirror pods.
+
+### Configuring mirroring peers
```

Comment:
How about moving this section higher in the doc before the list of settings? You need to read about how to use the feature before you need to know the detailed CRD settings. The same change would apply to the rbd doc